internal/analysis/model_test.go

```
@@ -337,6 +338,256 @@ func TestSnapshotHasNoExportedCollectionFields(t *testing.T) {
+	document.DisplayPath = "mutated.env"
+	document.Lines[0].Number = 99
+	document.Lines[0].Key = "MUTATED"
+	document.Lines = document.Lines[:1]
+
+	keys := inventory.Keys()
+	keys[0] = "MUTATED"
+	declarations := inventory.Declarations("TOKEN")
+	declarations[0].DocumentID = "mutated"
+	declarations[0].DisplayPath = "mutated.env"
+	declarations[0].LineNumber = 99
+	declarations = declarations[:0]
```

golangci-lint (2.13.2)

[error] SA4006: this value of declarations is never used

(staticcheck)